tools/src/main/python/dlpx/virtualization/_internal/delphix_client.py

```diff
@@ -20,10 +22,14 @@ class DelphixClient(object):
     """
     __BOUNDARY = '----------boundary------'
     __UPLOAD_CONTENT = 'multipart/form-data; boundary={}'.format(__BOUNDARY)
+    __WAIT_TIME_SECONDS = 5
+    __WAIT_TIMEOUT_SECONDS = 1800
     __cookie = None
 
-    def __init__(self, engine):
+    def __init__(self, engine, timeout=None):
         self.__engine = engine
+        if timeout is not None:
+            self.__WAIT_TIMEOUT_SECONDS = timeout
```

Comment:
Will these 2 variables total wait timeout and polling interval be available in the config file or need a separate change for that?

Reply:
Need a separate change for that. This is just used for tests.